contrib/src/templates/mod.rs

```diff
@@ -329,12 +374,13 @@ impl Template {
 /// rendering fails, an `Err` of `Status::InternalServerError` is returned.
 impl Responder<'static> for Template {
     fn respond_to(self, req: &Request) -> response::Result<'static> {
-        let ctxt = req.guard::<State<Context>>().succeeded().ok_or_else(|| {
+        let mc = req.guard::<State<ManagedContext>>().succeeded().ok_or_else(|| {
             error_!("Uninitialized template context: missing fairing.");
             info_!("To use templates, you must attach `Template::fairing()`.");
             info_!("See the `Template` documentation for more information.");
             Status::InternalServerError
         })?;
+        let ctxt = mc.get();
```

Comment:
Just add the `.get()` to the thing above.

Reply:
I will see what happens with other refactors and check this again, but I believe this was done because `ctxt` borrows from `mc`, so the expression assigned `mc` can't be a temporary.